### PR TITLE
use size instead of pages

### PR DESCRIPTION
**Pull request by Hakuyume**

### CODE REVIEW

**Review by S-Nakamur-a — APPROVED** [review]
LGTM!!!!!

File: src/main.rs [hunk 1]
```diff
@@ -70,10 +69,10 @@ fn main(handle: Handle, mut system_table: SystemTable<Boot>) -> Status {
...
         let kernel_buffer = fs_sytem_table
             .boot_services()
-            .allocate_pool(MemoryType::LOADER_DATA, kernel_file_size_in_pages)
+            .allocate_pool(MemoryType::LOADER_DATA, kernel_file_size)
             .unwrap();
 
-        let buffer = from_raw_parts_mut(kernel_buffer, kernel_file_size_in_pages * PAGE_UNIT_SIZE);
+        let buffer = from_raw_parts_mut(kernel_buffer, kernel_file_size);
         // ファイルサイズがでかいとこの行で落ちる
         // X64 Exception Type - 0E(#PF - Page-Fault)  CPU Apic ID - 00000000
```
Comment (S-Nakamur-a) [inline src/main.rs]:
```suggestion
```

**Review by makotoshimazu — APPROVED** [review]
LGTM!

**Review by yukidmy — APPROVED** [review]
LGTMLGTM

**Pull request merged**